### DISCUSSION TITLE
Grabaciones Semana 1 Misión FrontEnd

### DISCUSSION THREAD

**Discussion by romarpla** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#17] [Announcements]
Hola Explorers! @LaunchX-InnovaccionVirtual/mission-innova001 

Les dejo los links de las grabaciones de esta primer semana para que puedan verlas en caso de no haber estado en el live y también los links que nos compartió Kamilo en la Mision Station de UI/UX.

Live de bienvenida 14/02/22 --> https://web.microsoftstream.com/video/ef66ed9e-c981-41eb-98a6-35272b3c5b73
 
Live de intro FrontEnd 16/02/22 --> https://web.microsoftstream.com/video/d02d524e-8bbb-466e-acc1-13407fd12747
 
Live Mision Station UI/UX 17/02/22 --> https://web.microsoftstream.com/video/9bcebbbb-a07a-4ffe-a4b8-3e9ad0c6b5e8
 
Links de sesión de UI/UX
https://material.io/design
https://www.carbondesignsystem.com/
https://www.nngroup.com/articles/ten-usability-heuristics/
https://contrastchecker.com/
https://www.worlduiresources.com/
https://tablericons.com/
https://lordicon.com/
https://dribbble.com/
https://www.behance.net/
https://www.amazon.com/-/es/Steve-Krug/dp/0321344758
https://www.nngroup.com/articles/ten-usability-heuristics/
https://developer.apple.com/design/human-interface-guidelines/
https://coolors.co/222222-ffffff-4b4e6d-84dcc6-95a3b3
https://contrastchecker.com/
 
¡Vámonos hasta el espacio y más allá Explorers! 
RM

**Comment by Rob-Alc** [comment]
Muchas gracias!!
Excelente tarde!!

**Comment by Bestoff-77** [comment]
Buenas tardes. Aquí tengo una duda estas sesiones de que son perdón, ahí si ya me perdí. Me quedé en que teníamos hasta el lunes 21 para entregar lo de python y que ese mismo día era la apertura de front end.¿Qué me perdí? 🤔El 18 feb 2022 4:10 p. m., Romarpla ***@***.***> escribió:

Hola Explorers! @LaunchX-InnovaccionVirtual/mission-innova001

Les dejo los links de las grabaciones de esta primer semana para que puedan verlas en caso de no haber estado en el live y también los links que nos compartió Kamilo en la Mision Station de UI/UX.

Live de bienvenida 14/02/22 --> https://web.microsoftstream.com/video/ef66ed9e-c981-41eb-98a6-35272b3c5b73

Live de intro FrontEnd 16/02/22 --> https://web.microsoftstream.com/video/d02d524e-8bbb-466e-acc1-13407fd12747

Live Mision Station UI/UX 17/02/22 --> https://web.microsoftstream.com/video/9bcebbbb-a07a-4ffe-a4b8-3e9ad0c6b5e8

Links de sesión de UI/UX

https://material.io/design

https://www.carbondesignsystem.com/

https://www.nngroup.com/articles/ten-usability-heuristics/

https://contrastchecker.com/

https://www.worlduiresources.com/

https://tablericons.com/

https://lordicon.com/

https://dribbble.com/

https://www.behance.net/

https://www.amazon.com/-/es/Steve-Krug/dp/0321344758

¡Vámonos hasta el espacio y más allá Explorers!

RM

**Reply by manusanabria** [reply]
Si andas bastante perdido amigo, deberías apurarte a enviar las katas de python, las sesiones de frontend comenzaron esta semana. ;)

**Reply by vicogarcia16** [reply]
Creo que o no has entrado a las sesiones de FrontEnd o de plano estuviste ocupado lidiando con las katas (guiño, guiño) . No te preocupes, esa es la información que se ha generado de todas las sesiones de esta semana y algunos links que nos han compartido referentes a la introducción del módulo FrontEnd. 
_**Te sugiero leer y repasar todo lo que se ha visto (todo el chorizo visto arriba)**_. Por tanto, lo contenido en este repositorio para que vayas a la par de los démas. Ánimo :3

**Reply by JosePabloGrr** [reply]
**Había dos fechas de entrega para el OnBoarding**: Las katas de Python las podías entregar el día **Lunes 14 de Febrero** y así obtener acceso a la Semana 1 - Despegue de FrontEnd.
Si no tuviste tiempo de entregarlas todavía tienes hasta este próximo **Lunes 21 de Febrero** para entregar tus katas e ingresar a las grabaciones de la Semana 1 y continuar con el curso de manera regular la semana entrante, una vez con tu acceso a FrontEnd en Teams, veras las fechas que están impartiendo los Lives y los demás enlaces.

**Reply by JuanSalvador73** [reply]
El día lunes 21 fue la fecha límite para la entrega de Katas. Como sea, si tuviste algún problema puedes escribir a la cuenta de correo LaunchX para explicar posibles retrasos. 

Entrega las Katas que puedas e incríbete cuanto antes enviando el link de las mismas para que entres a la misión FrontEnd, aún estás a tiempo. Saludos y suerte.

**Comment by RogelioMezaL** [comment]
Hasta cuando tengo para poder enviar las katas de Python?


Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: Manuel ***@***.***>
Enviado: viernes, 18 de febrero de 2022 06:30 p. m.
Para: ***@***.***>
CC: ***@***.***>; Team ***@***.***>
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Grabaciones Semana 1 Misión FrontEnd (Discussion #17)


Si andas bastante perdido amigo, deberías apurarte a enviar las katas de python, las sesiones de frontend comenzaron esta semana. ;)

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208214>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXWEH6DGQXDDYML6VDXWAG3U33QDHANCNFSM5OZGOQIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by JosePabloGrr** [reply]
https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208221

**Reply by JorgeSoberanis** [reply]
Buenas noches compañero, antes que nada espero que estés bien. Y resolviendo tu duda, las katas tiene como fecha limite el día lunes 21 de este mes, asi que aun estas a tiempo :D

**Comment by DanielDuran1009** [comment]
Exactamente, según yo hoy es último día para las katas de Python 12:00
medianoche cierran y no las aceptan, ánimo compañero las 8 y 9 están duras
(en mi opinión)

**Comment by Bestoff-77** [comment]
Si he checando las grabaciones 😆, y entendí eso, que el mismo lunes se abría front, pero bueno, les agradezco la aclaración y pues a presionarme porque ya solo tengo 2 días y me faltan katas 😅. El 18 feb 2022 6:28 p. m., "José G. Rivera" ***@***.***> escribió:

Había dos fechas de entrega para el OnBoarding: Las katas de Python las podías entregar el día Lunes 14 de Febrero y así obtener acceso a la Semana 1 - Despegue de FrontEnd.

Si no tuviste tiempo de entregarlas todavía tienes hasta este próximo Lunes 21 de Febrero para entregar tus katas e ingresar a las grabaciones de la Semana 1 y continuar con el curso de manera regular la semana entrante, una vez con tu acceso a FrontEnd en Teams, veras las fechas que están impartiendo los Lives y los demás enlaces.

**Comment by HectorMiguel71** [comment]


**Reply by rojaslcc** [reply]
Al entregar las katas de python debes estar logeado eb office.com con tu correo de innovacción. El form donde entregas las katas solo debes llenarlo y si todo sale bien, el sistema en automático te mando un correo de confirmación a tu correo de innovacción pero en Teams en automático el sistema te da acceso al Grupo de Mission FrontEnd.

**Reply by JosePabloGrr** [reply]
Envíate tus Katas y Obtuviste el correo de ingreso satisfactorio a Frontend? Asumo que si, por lo tanto..
1. Ingresa a tu Teams, del lado izquierdo selecciona "Equipos"
2. Seleccionar equipo "Misión Frontend - JS" * Si no lo ves, es por que no enviaste bien tu formulario de katas *
3. Abajo del titulo Mision Frontend.. seleccionas "Semana 1 - Despegue" * _Si no te aparece seguro esta abajo de General, presiona el botón de ver más._
4. En el contenido central veras el resumen de los días anteriores y un post llamado "Grabaciones" con todos los enlaces. Confirmo que están activos.
![image](https://user-images.githubusercontent.com/16726985/154778966-1868aa0c-2e5f-47be-b485-eab7937257b8.png)

**Reply by HectorMiguel71** [reply]
No me llegó ni la invitación de lo de phyton ¿en dónde pongo las katas?

**Reply by JorgeLMarquez** [reply]
Hola Héctor yo tampoco podía enviarlas porque tenia otro correo de teams, te sugiero que abras un navegador en incógnito y ahí mismo inicies sesión para abrir el formulario

**Reply by HectorMiguel71** [reply]
¿No me dieron correo de innovación? ¿Cómo se consigue? sólo me llegó esta invitación
![image](https://user-images.githubusercontent.com/99772437/154779964-22d9537f-088d-4827-9cc1-6c3faecafc0a.png)

Según esto tenía entendido que el 15 iniciaba todo ¿Cómo puedo ponerme al día?

**Comment by JoseSanchezMendez** [comment]
Hasta el 21

**Reply by HectorMiguel71** [reply]
¿A que correo puedo enviarlas?

**Reply by PabloDoria** [reply]
Hola Hector, no se mandan a un correo, debes hacer un repositorio personal y subir allí tus Katas, copias la URL de dicho repositorio, luego vas al canal general de Teams y bajas hasta la publicación de la imagen, ahí la MC da mas indicaciones, lo mas importante es que la URL esté bien y que accedas al formulario con tu correo de Launch X/Inovacción virtual.

![image](https://user-images.githubusercontent.com/87738319/154786106-aaddff49-cc85-442a-a21f-eba5da566e0b.png)


En la sesión del viernes 11 se dieron las indicaciones completas para la forma de entrega paso a paso, te recomiendo que lo veas, el enlace directo está en el canal general, en la barra de arriba eliges primera misión y bajas, está justo debajo de las Katas.

![image](https://user-images.githubusercontent.com/87738319/154786040-f1c85b78-cd95-4841-922e-80b3a34eb9c6.png)

**Reply by HectorMiguel71** [reply]
El detalle es que no tengo equipos en teams.

**Reply by PabloDoria** [reply]
¿Ya entraste a Teams con tu correo de Innovacción Virtual, verdad? Si es así entonces debe ser problema de la organización, contacta con ellos por Instagram o por el repositorio de Python, la Sherpa revisa todo el contenido de discusiones. Tengo entendido que solo se pueden subir las Katas por el formulario porque está automatizado

**Reply by HectorMiguel71** [reply]
Pablo, no me dieron el correo de Innovación solo me llegó éste correo
![image](https://user-images.githubusercontent.com/99772437/154787358-ab5ddec8-a667-4993-8ded-66594e08dc32.png)

**Comment by hectorcastro1996** [comment]
Chale, es muy pesado hacer esto entre semana, apenas puedo los fines y los
accesos me los dieron el lunes :c

**Comment by marytcg03** [comment]
Por favor no estén respondiendo a todos, lo importante se pierde entre el spam , las fechas y la información importante esta en teams. Se los digo de buena manera compañeros. :D

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: ***@***.***>
Enviado: viernes, 18 de febrero de 2022 05:46 p. m.
Para: ***@***.***>
CC: MARIANA ***@***.***>; Team ***@***.***>
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Grabaciones Semana 1 Misión FrontEnd (Discussion #17)

Chale, es muy pesado hacer esto entre semana, apenas puedo los fines y los
accesos me los dieron el lunes :c

El vie., 18 de febrero de 2022 5:42 p. m., Daniel Duran <
***@***.***> escribió:

> Exactamente, según yo hoy es último día para las katas de Python 12:00
> medianoche cierran y no las aceptan, ánimo compañero las 8 y 9 están duras
> (en mi opinión)
>
> El El vie, feb. 18, 2022 a la(s) 6:31 p.m., Manuel Sanabria <
> ***@***.***> escribió:
>
> > Si andas bastante perdido amigo, deberías apurarte a enviar las katas de
> > python, las sesiones de frontend comenzaron esta semana. ;)
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208214
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AXXCNMMAOER3JK75AB4RF73U33QEVANCNFSM5OZGOQIA
> >
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID:
> >
> <LaunchX-InnovaccionVirtual/FrontEnd-Mision/repo-discussions/17/comments/2208214
> > @github.com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208248>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXY7BQ7JBXJSRQZB7JTLBD3U33RPXANCNFSM5OZGOQIA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are on a team that was mentioned.Message
> ID:
> <LaunchX-InnovaccionVirtual/FrontEnd-Mision/repo-discussions/17/comments/2208248
> @github.com>
>


—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208275>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AH4SRMFTUNWKYZJEQOQUBOLU33R5NANCNFSM5OZGOQIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by JoselineGomezV** [comment]
Hola, buenas noches

creo que ya no puedes enviar las katas de Pyhton. Tengo entendido que se tenia hasta hoy viernes, pero el formulario para entregar las katas se cerro desde las 12:00 am.

Desconozco si hay otro link para enviar las katas de Python.

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: Humberto Rojas ***@***.***>
Enviado: viernes, 18 de febrero de 2022 06:48 p. m.
Para: ***@***.***>
CC: Joseline Del Carmen Fiz ***@***.***>; Team ***@***.***>
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Grabaciones Semana 1 Misión FrontEnd (Discussion #17)


Al entregar las katas de python debes estar logeado eb office.com con tu correo de innovacción. El form donde entregas las katas solo debes llenarlo y si todo sale bien, el sistema en automático te mando un correo de confirmación a tu correo de innovacción pero en Teams en automático el sistema te da acceso al Grupo de Mission FrontEnd.

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208271>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVHPQK4XXRECA5R4EELE2YLU33SDZANCNFSM5OZGOQIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by OcielOrtegaVazquez** [comment]
Otra vez comentarios aquí ...... por favor realizarlos en los respectivos apartados. 
Saludos cordiales

**Reply by UrielCanchola** [reply]
Discussions es el apartado ideal, para no recibir correos solo desactiva las notificaciones y listo no es dificil.

**Comment by hugocastp** [comment]
https://forms.office.com/pages/responsepage.aspx?id=BUvlSn63JEKu8YZhQi4IFl_tYfpu3chInD3flo8jTZhUNlYwT09ES0tIUUhKRjA3VDhOU0VQUjBTNi4u


Este formulario no sirve?

**Comment by luisrosey** [comment]
Por favor, dejen de responder a todos y hacer spam, en buena onda si tienen dudas mándele mensaje al Instagram del curso o manden un mail directo a ellos. Gracias.

On 18 Feb 2022, at 18:58, JoselineGomezV ***@***.***> wrote:

﻿

Hola, buenas noches

creo que ya no puedes enviar las katas de Pyhton. Tengo entendido que se tenia hasta hoy viernes, pero el formulario para entregar las katas se cerro desde las 12:00 am.

Desconozco si hay otro link para enviar las katas de Python.

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: Humberto Rojas ***@***.***>
Enviado: viernes, 18 de febrero de 2022 06:48 p. m.
Para: ***@***.***>
CC: Joseline Del Carmen Fiz ***@***.***>; Team ***@***.***>
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Grabaciones Semana 1 Misión FrontEnd (Discussion #17)


Al entregar las katas de python debes estar logeado eb office.com con tu correo de innovacción. El form donde entregas las katas solo debes llenarlo y si todo sale bien, el sistema en automático te mando un correo de confirmación a tu correo de innovacción pero en Teams en automático el sistema te da acceso al Grupo de Mission FrontEnd.

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208271>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVHPQK4XXRECA5R4EELE2YLU33SDZANCNFSM5OZGOQIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you are on a team that was mentioned.Message ID: ***@***.***>


—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2208286>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXU62PZRX3I4CRLHSJCZCCDU33TKVANCNFSM5OZGOQIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by MesachVenegas** [comment]
Excelente muchas gracias por los links

**Comment by MARYCHS** [comment]
Mil gracias x compartirlas

**Comment by JonathanGarcia15** [comment]
NO DEJAN DE LLEGARME CORREOS, ¿POR QUÉ ME MANDAN SPAM DE ESTA COSA? Ya sé cuáles son los enlaces y ví 1 en vivo, no había necesidad de subscribir a todos. OMG

**Reply by franciscoxcode** [reply]
Pues tú tienes que configurar en tu cuenta qué tipo de correos recibir y cuáles no.

**Comment by FMartinRG** [comment]
Muchas gracias por la confirmación buena noche!!

**Comment by ferhdz95** [comment]
Hola Uriel buenas noches, por favor cuando respondas un correo dandole click en responder, no le des click en responder a todos para poder evitar el spam. Saludos.

Get Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by retroskateboard** [comment]
Dejen de hacer SPAM por favor, dudas directamente a los correos de los lideres y dejen dormir.

**Comment by MARYCHS** [comment]
Hola,

Donde obengo la liga para la sesión de hoy.

Saludos y gracias

**Reply by iRetr0o** [reply]
Está en la semana 2 que se llama primer vuelo 🛸

**Comment by therockvan** [comment]
Muy buenos días, ya tengo acceso a la misión FrontEnd el domingo 20 hice mi
form en el cual me dieron acceso mis katas ya se encuentran en mi usuario
de Git, tarde por que no implemente bien los comandos para subir mis catas,
ya al revisar bien la información que nos dio la MC Fernanda Ochoa ya fue
exitoso el subir citada información, actualmente me encuentro iniciando los
textos de introducción a FrontEnd y seguir con el proceso de aprendizaje,
agradezco la atención prestada al presente

**Comment by joseissai** [comment]
Buenas tardes yo tengo mis katas pero me confundí para el envío, cuál es el
correo de LaunchX para poder enviarlas?



Saludos.

**Reply by DanielGalindoRios** [reply]
Hola.

 las katas no se envian,  Debes crear un repositorio en GitHUb ( puedes ver el video parte dos de github ) en el onboarding:

https://www.youtube.com/watch?v=8B_qtbdlLSU


Despues en el formularion de Tams pones la liga de tu repositorio de las katas y automaticamente te da acceso al nuevo grupo en teams.

Saludos

**Comment by joseissai** [comment]
Excelente muchas gracias ya las tengo en mi repositorio, deja hago el
enlace.


Saludos.

**Comment by choreweb** [comment]
Buen dia,

Estuve algo ocupado dias previos y apenas pude hacer la mitad de las katas... https://github.com/choreweb/CursoIntroPython
[https://opengraph.githubassets.com/3c05b5e1b76d81c7e3d2a4d36d9085cd5db10f09509567d652c7ff4a4bed08aa/choreweb/CursoIntroPython]<https://github.com/choreweb/CursoIntroPython>
GitHub - choreweb/CursoIntroPython<https://github.com/choreweb/CursoIntroPython>
Contribute to choreweb/CursoIntroPython development by creating an account on GitHub.
github.com

Espero pueda acceder aun a la parte front.

Ya el forms esta cerrado (por eso mando el link por aqui)

Gracias de antemano

Saludos.

-Eliezer Elizondo
________________________________
De: Juan Salvador ***@***.***>
Enviado: miércoles, 23 de febrero de 2022 01:14 a. m.
Para: LaunchX-InnovaccionVirtual/FrontEnd-Mision ***@***.***>
CC: choreweb ***@***.***>; Team mention ***@***.***>
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Grabaciones Semana 1 Misión FrontEnd (Discussion #17)


El día lunes 21 fue la fecha límite para la entrega de Katas. Como sea, si tuviste algún problema puedes escribir a la cuenta de correo LaunchX para explicar posibles retrasos.

Entrega las Katas que puedas e incríbete cuanto antes enviando el link de las mismas para que entres a la misión FrontEnd, aún estás a tiempo. Saludos y suerte.

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/17#discussioncomment-2234086>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUAOCDAHWGX3OWZ4EKW6CYDU4SCPBANCNFSM5OZGOQIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by EddAntMar** [reply]
Hola, si obtuviste respuesta por parte de Innovaccion y te dieran acceso al curso de front? 

Yo también subí mi repo con las katas y les envié correos con la solicitud de acceso pero no obtuve respuesta, no se si escribirles por instagram o algún otro medio.

Saludos.

**Reply by carlos-gmx3** [reply]
No es como q tenga acceso, este es un repositorio público. Ya el acceso al grupo de Teams es otra cosa, ese si era llenando el formulario para entregar las katas

**Comment by choreweb** [comment]
No me respondieron nada... mandé correo a contacto@ (su dominio de correo que no recuerdo) pero pues tampoco...

Si llegas a saber algo, te agradecería me escribieras.

Saludos

-Eliézer

Enviado desde mi Samsung Mobile de Telcel
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by LTWisi** [comment]
La fecha límite fue la semana pasada. Ya se perdió su oportunidad...

**Comment by ArielCanedo** [comment]
Exacto, podías subir las katas que tuvieras, se dijo en los lives y como
las tuvieras

**Comment by choreweb** [comment]
Si, de hecho hace rato vi que está el curso cargado, igual se podría ver directo y hacer los ejercicios...  Ya si hay dudas pues investigar aparte...

Enviado desde mi Samsung Mobile de Telcel
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Andres1380** [comment]
Buenas noches.
¿Qué se puede hacer en mi caso? Hace apenas una semana pude acceder a mi cuenta porque me dieron mi contraseña mal. Mandé un correo y nunca me respondieron. Solo pude acceder porque un compañero que me quiso ayudar, por pura suerte se le ocurrió probar cambiando el último dígito, y funcionó.

Por otras actividades apenas he tenido chance de hacer dos katas, y no sabía que tenía de límite hasta el viernes.

**Reply by carlos-gmx3** [reply]
Esperar a la siguiente convocatoria o a q te respondan de launch x
